blockmanager.go

```diff
@@ -82,7 +82,7 @@ type cfheadersMsg struct {
 type processCFHeadersMsg struct {
 	earliestNode *headerNode
 	stopHash     chainhash.Hash
-	extended     bool
+	filterType   uint8
```

Comment:
Perhaps we should introduce an enum here? We can define it within `btcsuite.wire`. That way it cna be re-used across all the contexts where it would be referenced: `btcd`, `neutrino`, `btcutil`, etc. 

The upside of this is that we also get rid of all the seemingly magic numbers, so we can say:
```
filterType == wire.GCSFilterRegular  (or w/e other name)
```

instead of:
```
filterType == 1
```

Reply:
I've updated this to use `wire.FilterType`. We might consider eventually moving to that type throughout instead of `filterdb.FilterType` as well, but that's not urgent IMO.